### PR TITLE
HLM-722 | Creating new tenant-portal app

### PR DESCRIPTION
**Pull request by saiprakash-v**
@prasanna-egov @vishwajeet-eGov

### CODE REVIEW

**Review by vishwajeet-eGov — APPROVED** [review]

File: tenant-portal/README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```
Comment (prasanna-egov) [inline tenant-portal/README.md:1]: Modify the Readme. Dont use the default readme.

File: tenant-portal/src/App.css [hunk 2]
```diff
@@ -0,0 +1,38 @@
+.App {
```
Comment (prasanna-egov) [inline tenant-portal/src/App.css:1]: This is default App.css Change this please

File: tenant-portal/src/components/CreateSchema/CreateSchema.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import axios from 'axios';
+import { useEffect } from "react";
+import { useKeycloak } from "@react-keycloak/web";
+
+function CreateSchema() {
+    const { keycloak } = useKeycloak();
+
+    const config = {
```
Comment (prasanna-egov) [inline tenant-portal/src/components/CreateSchema/CreateSchema.js:9]: Use axios interceptor. Don't set header for every request.

File: tenant-portal/package.json [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^13.5.0",
+    "axios": "^0.27.2",
+    "bootstrap": "^5.2.1",
+    "keycloak-js": "^19.0.2",
+    "react": "^18.2.0",
+    "react-bootstrap": "^2.5.0",
+    "react-dom": "^18.2.0",
+    "react-router-dom": "^5.2.0",
```
Comment (prasanna-egov) [inline tenant-portal/package.json:16]: react router 6 is released right. Please upgrade to that.

**Review by prasanna-egov — APPROVED** [review]

**Pull request merged**